### PR TITLE
Add deprecation of money_format

### PR DESCRIPTION
**Pull request by BackEndTea**
I hope this is correct, i copy pasted this from other functions

### CODE REVIEW

File: reference/strings/functions/money-format.xml [hunk 1]
```diff
@@ -229,6 +233,30 @@
...
+      <row>
+       <entry>&Version;</entry>
+       <entry>&Description;</entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>7.4.0</entry>
+       <entry>
+        This function has been deprecated.
```
Comment (nikic) [inline reference/strings/functions/money-format.xml]: I think we should give some indication as to what you should use instead -- in this case that would be the number formatting functionality in intl.

File: reference/strings/functions/money-format.xml [hunk 2]
```diff
@@ -229,6 +233,32 @@
...
+       <entry>&Version;</entry>
+       <entry>&Description;</entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>7.4.0</entry>
+       <entry>
+        This function has been deprecated. Instead, 
+        use <function>numberformatter.formatcurrency</function>.
```
Comment (nikic) [inline reference/strings/functions/money-format.xml]: I believe the correct syntax is `<methodname>NumberFormatter::formatCurrency</methodname>`

File: reference/strings/functions/money-format.xml [hunk 3]
```diff
@@ -229,6 +233,32 @@
...
+       </entry>
+       </entry>
```
Comment (nikic) [inline reference/strings/functions/money-format.xml]: There is a stray `</entry>` here.

**Comment by nikic** [comment]
Small ping, did you see my last two comments?

**Comment by BackEndTea** [comment]
I saw them, and then comletely forgot 😅. I fixed them now

**Comment by cmb69** [comment]
Thanks! Applied as http://svn.php.net/viewvc?view=revision&revision=349216.

**Comment by BackEndTea** [comment]
Thanks @cmb69 & @nikic

**Pull request closed without merging**